cacli/cmd/cancel/cancel.go

```diff
@@ -0,0 +1,49 @@
+package cancel
+
+import (
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/cloud-annotations/training/cacli/cmd/login"
+	"github.com/cloud-annotations/training/cacli/e"
+	"github.com/jedib0t/go-pretty/text"
+	"github.com/spf13/cobra"
+)
+
+func Run(cmd *cobra.Command, args []string) {
+	if len(args) < 1 {
+		e.Exit(errors.New("no `Model ID` provided\nUsage: cacli cancel <model_id>"))
+	}
+
+	modelID := args[0]
+
+	session := login.AssertLoggedIn()
+
+	model, err := session.GetTrainingRun(modelID)
+	if err != nil {
+		e.Exit(err)
+	}
+
+	switch model.Entity.Status.State {
+	case "completed", "error", "failed", "canceled":
+		fmt.Println("training status : ", model.Entity.Status.State)
+		e.Exit(err)
```

Comment:
there is no `err` here, this should be 
```
e.Exit(errors.New(fmt.Sprintf("can not cancel an already %s training run", model.Entity.Status.State)))
```

however, this sounds a bit strange:
"can not cancel an already **error** training run"

so maybe break this into 2 cases:
```
case "completed", "canceled":
		e.Exit(errors.New(fmt.Sprintf("can not cancel an already %s training run", model.Entity.Status.State)))
case "error", "failed":
		e.Exit(errors.New("can not cancel an already failed training run"))
```